transformer.md

```diff
@@ -255,3 +255,62 @@ The end marker is the following, where _`<N>`_ is the number of the step:
 ```markdown
 <!-- Killercoda step<N>.md END -->
 ```
+
+## Generate a tutorial
+
+You can generate a new tutorial from existing documentation using the transformer tool.
+After generation, a [GitHub Actions workflow](./.github/workflows/regenerate-tutorials) updates the tutorial when the documentation source changes.
+
+### Before you begin
+
+1. Clone the repository with the source documentation.
+1. Clone the Killercoda repository.
+1. [Download and install Go](https://go.dev/doc/install) to run the transformer tool.
+
+To generate a tutorial:
+
+1. In the source repository, check out a new branch.
+
+   Give the branch a name that reflects the planned changes.
+   For example, `2024-06-killercoda-tutorial-for-loki-quickstart`.
+
+1. In the Killercoda repository, check out a new branch.
+
+   For convenience, name the branch the same as you did in the source repository.
+
+1. In the source repository, run `make docs` from the `docs/` directory so that you can verify your changes don't break the rendered documentation.
+
+1. In the source file, add the Killercoda front matter.
+
+   For more information and an example front matter, refer to [Metadata](#Metadata)
+
+1. Add directives for each step in the tutorial.
+
+   Each step starts at the [step](#step) directive start marker and ends at the [step](#step) directive end marker.
```

Comment:
Without confusing it, is worth mentioning they can add a start and finish page aswell with this notation?

Reply:
Yeah good shout, I'll add each as a step and explain that you can either include start and finish directives or set that in the front matter.

Reply:
I've added "Configure an introduction page" and "Configure a finish page" steps before and after that discuss the two options (page in the in the Killercoda repo or page from the source file).